content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,65 @@
+
+Today, we're announcing that you can now run [batch changes server-side (beta)](https://docs.sourcegraph.com/batch_changes/explanations/server_side). 
+
+Setting up running batch changes server-side allows to:
+- Run large-scale or resource intensive batch changes without clogging your local machine.
+- Run large batch changes faster by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience, with logs being streamed directly into Sourcegraph.
+- Trade-off a little more setup for the Sourcegraph admin, against less setup for users (no local `src-cli`)
+
+This is the outcome of months of iteration on the experimental version of the feature, and we're grateful to early users for their feedback!
+
+This feature requires to setup [executors](https://docs.sourcegraph.com/admin/executors) (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run Code Intelligence [auto-indexing](https://docs.sourcegraph.com/code_intelligence/how-to/enable_auto_indexing) (experimental).
+
+<!-- TODO: asset welcome right after deployment on demo -->
+
+##  Batch specs can now mount local files on steps containers (experimental)
+
+As we improve Batch Changes, we're heavily focusing on making it easy to write and debug batch specs. Iteration time needs to be as small as possible, so that users can quickly put together a spec, run it, see results, and quickly improve on it.
+
+Many users have custom tooling or scripts that they run as batch changes steps. The only way to do that until now was to bake those scripts into the step container, or to copy paste the scripts directly in the spec using [steps.files](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#steps-files). Both options slow down iterating on batch changes, requiring either to rebuild the container at each modification, or to copy paste code inside YAML.
+
+Now you can directly mount files or directories on the batch change container using [steps.mount](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#steps-mount). This is an experimental feature that's only supported when running batch changes locally. Lots of improvements are on the way ([issue](https://github.com/sourcegraph/sourcegraph/issues/14851)).
+
+We're committed to making the process of iterating on batch changes as fast as possible. Submit feedback through your Sourcegraph account manager, or raise an issue [here](https://github.com/sourcegraph/sourcegraph/issues/new?title=Batch%20changes:&body=@batchers%20this%20is%20an%20issue%20for%20you)!
+
+## Sort and filter "detect and track patterns" code insights series 
```

Comment:
@Joelkw - should this be Sort and filter "detect and track patterns" in Code Insights series 

(I added "in" and capitalized Code Insights).

Reply:
No, no "in," these are the types of insights/series, but good question. Not sure about capitalization, I think lower case since it's a product detail but no strong pref